cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -164,6 +164,7 @@ protected void configure() {
                                bind(MRJobInfoFetcher.class).to(LocalMRJobInfoFetcher.class);
                                bind(StorageProviderNamespaceAdmin.class).to(LocalStorageProviderNamespaceAdmin.class);
                                bind(UGIProvider.class).to(UnsupportedUGIProvider.class);
+                               bind(OwnerAdmin.class).to(DefaultOwnerAdmin.class);
```

Comment:
I don't think you need to bind it here. This binding is configure from which in-mem, standalone, distributed is constructed.